### ISSUE TITLE
Potential Error in documentation of Reock and ConvexHull compactness measures

### ISSUE THREAD

**Issue by madeleinegoertz** [kosukeimai/redist#44]
Hello,

I've been using your library to simulate redistricting in Virginia, and I believe I've found a small error in your documentation. `redist.compactness()` computes, among many other metrics, both a `Reock` and a `ConvexHull` score. Both in the literature and in your source code, the `Reock` score is the ratio of the area of the districts to the minimum bounding circle, and the `ConvexHull` score is the ratio of the area of the district to the are of the minimum convex hull. 

However, the [documentation ](http://kosukeimai.github.io/redist/reference/redist.compactness.html) for this function omits an explanation of the `ConvexHull` score in the "Details" section and (I believe) incorrectly describes the `Reock` score as computing the `ConvexHull` score, as seen below. 

![image](https://user-images.githubusercontent.com/22863903/111104236-75103e80-850d-11eb-9214-6ba5c07b77bd.png)

In summary, I think this is just a small error of omission in the documentation w.r.t. `ConvexHull` and an erroneous explanation of `Reock`. 

Thanks!

**Comment by christopherkenny** [comment]
Thank you for pointing this out. The definitions should now be correct and include a quick note on value interpretation. The website should update soon to reflect that.
